tests/lib/rules/no-invalid-html-attribute.js

```diff
@@ -130,6 +130,9 @@ ruleTester.run('no-invalid-html-attribute', rule, {
     { code: '<link rel="icon"></link>' },
     { code: 'React.createElement("link", { rel: "icon" })' },
     { code: 'React.createElement("link", { rel: ["icon"] })' },
+    { code: '<link rel="shortcut icon"></link>' },
+    { code: 'React.createElement("link", { rel: "shortcut icon" })' },
+    { code: 'React.createElement("link", { rel: ["shortcut icon"] })' },
```

Comment:
these should be valid, but shouldn't `shortcut` and `shortcut foo` and `foo shortcut` all be invalid?

Reply:
Yep but I am having a hard time wrapping my head around the code and the error messages.

So normally `neverValid` would trigger - which is kind of not true - because paired with `icon` it is valid on link.
I changed the Map item to `['shortcut\u0020icon', new Set(['link'])],` which also doesn't help because then it still reports `shortcut` to be never valid and I don't want to allow `shortcut` to be valid alone 🤔

Reply:
since this is a special case of two words, we might need to alter the rule where it splits on a space.

Reply:
Not feeling super comfortable doing that on my first contribution to this project though 😅  Wouldn't really know where to start and keep it clean.

Reply:
That’s what code review is for, if you’re up for it :-)

Reply:
Alright I'll take a stab at it 🔪

Reply:
OK did some progress, need to take another look regarding multiple spaces, signing off for today 😄